chatbot_train/unittest_train.py

```diff
@@ -21,56 +21,48 @@ def setUpClass(self):
         
  
     def test_default_parms(self):
-            self.assertTrue(train_run())
+            #self.assertTrue(train_run())
             train = train_run()[1]
-            try:
-               self.assertAlmostEqual(train,25,places=1)
-            except AssertionError:
-                pass
+            self.assertNotAlmostEqual(train,0.0)
+            
+        
 
     def test_with_parms(self):
         #setup data
         data = self.test_cfg["data1"]
         epochs = self.test_cfg["epochs1"]
         #test with data and epochs values
         train = train_run(data,epochs)[1]
-        try:
-           self.assertAlmostEqual(train,25,places=1)
-        except AssertionError:
-            pass
-        self.assertTrue(train_run(data,epochs))
+        self.assertNotAlmostEqual(train,0.0)
```

Comment:
Two issues remaining:

Why comparing to 0.0? 

As discussed,  train_run returns a tuple:
``` python:
 return train_loss, train_accuracy, test_accuracy
```

Hence you will need to compare this tuple